R/method-differentialAbundance.R

```diff
@@ -78,7 +78,9 @@ cleanComparatorVariable <- function(data, comparator, verbose = c(TRUE, FALSE))
 
 DifferentialAbundanceResult <- setClass("DifferentialAbundanceResult", representation(
     effectSizeLabel = 'character',
-    statistics = 'data.frame'
+    statistics = 'data.frame',
+    pValueFloor = 'numeric',
+    adjustedPValueFloor = 'numeric'
```

Comment:
if were going this route, we could also provide a label for the adjusted pvalue, indicating what method was used? i dont actually remember, but am just kind of assuming wed like to be able to put 'FDR' somewhere frontend without having to hardcode it...

if youd like you could even group labels together under statsLabels and the other under statsMeta or something?

not essential.